networking/multiple_networks/primary_networks/about-user-defined-networks.adoc

```diff
@@ -24,6 +24,10 @@ Support for the Localnet topology on both primary and secondary networks will be
 
 Unlike a network attachment definition (NAD), which is only namespaced scope, a cluster administrator can use a UDN to create and define additional networks that span multiple namespaces at the cluster level by leveraging the `ClusterUserDefinedNetwork` custom resource (CR). Additionally, a cluster administrator or a cluster user can use a UDN to define additional networks at the namespace level with the `UserDefinedNetwork` CR. 
 
+The following diagram shows tenant isolation where each tenant has a single assigned UDN and uses the `ClusterUserDefinedNetwork` CR to define the UDN across multiple namespaces. A tenant acts as a disconnected network that is isolated from other tenants. Pods from a namespace in a tenant can communicate with pods in another namespace that exists in the same tenant.
```

Comment:
```suggestion
The following diagram shows tenant isolation where each tenant has a CUDN CR to allow the network to span across multiple namespaces that belong to each tenant. A tenant acts as a disconnected network that is isolated from other tenants (networks)?. Pods from a namespace in a tenant can communicate with pods in another namespace that exists in the same tenant.
```
its 1 CUDN that makes ns1 ns2 part of udn-1 and similarly for the second tenant. so 2 CUDNS... suggesting the removal of assigned UDN using a CUDN phrase there to avoid confusion.